transform/mattermost-analytics/models/staging/mm_telemetry_rc/stg_mm_telemetry_rc__performance_events.sql

```diff
@@ -0,0 +1,35 @@
+{{
+config({
+ "tags":"hourly",
```

Comment:
do we still need this if we've set it globally in the `dbt_project.yml` file?

Reply:
No, it can be removed.